computational-nuclear-activity.ipynb

```diff
@@ -284,7 +338,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The suggested solutions to these activities can be found in the `solution.py` script (along with answers to the discussion questions where applicable)"
+    "Test your function by calculating the activity of a source with initial activity of $10^{15}$ particles/second and decay constant of $\\lambda = 10^{-3}$ seconds$^{-1}$ that has passed through a shield of thickness 4 cm with a macroscopic cross section of $\\Sigma = 0.2$ cm$^{-1}$ and is measured at a distance of 1000 cm after 8000 seconds."
```

Comment:
I think they should probably be given an initial concentration rather than activity here if their solution to the first section uses concentration as an input.

Reply:
yes good idea